test/integration/tasks.test.js

```diff
@@ -195,6 +199,19 @@ describe('Tasks', function () {
   })
 
   describe('PATCH /tasks', function () {
+    before(async function () {
+      const user = {
+        first_name: 'Prakash',
+        last_name: 'C',
+        yoe: 0,
+        img: './img.png',
+        github_id: 'prakashchoudhary07',
+        username: 'sumit'
+      }
+      // Adding user
+      await userModel.addOrUpdate(user)
+    })
+
```

Comment:
There is a test below with ownerId = 'sumit' to update task. It would fail now after adding all the coversions.

Reply:
Got it, we could simply use the `addUser` util function in that case. 
Also, we need not pass other fields, only `username` should be sufficient

Reply:
First I tried using addUser. It didn't work for me. Then I found this example in other test and used it. I wasn't sure about required/optional fields. So just used the exact copy.

Reply:
Just pass this `user` object as an argument to the `addUser` function
Check the function definition for more info: https://github.com/Real-Dev-Squad/website-backend/blob/develop/test/utils/addUser.js

Reply:
Tried again. `addUser` does not work for input arguments. That function seems unclear to me. But I am hesitant to make a change there since it is being used at many places. I am referring to this line. It is overwriting the input argument into a boolean

`user = user && Object.keys(user).length === 0 && user.constructor === Object`

Reply:
I have asked ankur on dev-backend, waiting for his reply on the same

Reply:
You can go ahead with model function, no problem 👍🏻